tools/scripts/track-pr

```diff
@@ -95,17 +110,70 @@ def current_project_id() -> str:
 
 
 def cherry_pick_pr(pr_id: str) -> None:
-    commit = gql.get_pr_merge_commit(id=pr_id)["node"]["mergeCommit"]["oid"]
-    print(f"Cherry-picking {commit}")
+    if "determined-ee" in run_capture("git", "remote", "get-url", CLONED_REMOTE):
+        # TODO Add the separate logic for running EE-only cherry-picks.
+        print("Skipping EE PR, marking as conflicted for human attention")
+        add_item_to_project(current_project_id(), pr_id, FIX_CONFLICT_STATUS)
+        return
+
+    pr_commit = gql.get_pr_merge_commit(id=pr_id)["node"]["mergeCommit"]["oid"]
+    print(f"Cherry-picking {pr_commit}")
 
     try:
-        # TODO Actually run Git to do the cherry-pick.
-        status = NEEDS_TESTING_STATUS
+        # Find and fetch the PR commit and both release branches.
+        release_branch = max(
+            line.split()[1]
+            for line in run_capture(
+                "git", "ls-remote", CLONED_REMOTE, "refs/heads/release-*"
+            ).splitlines()
+        )[len("refs/heads/") :]
```

Comment:
Does `max()` actually do the right thing here?  I guess a lexical comparison is ok, but since it's a string containing a number (`refs/heads/release-0.20.0`) I worry a little bit about 0.100.0 being less than 0.99.0, for example.

Reply:
```python
>>> max(["0.100.0", "0.99.0"])
'0.99.0'
```

Reply:
Yeah, I just totally dashed this off without thinking it through. But it'll work for a while, so I'll fix it up in the next update.